### ISSUE TITLE
Looks interesting

### ISSUE THREAD

**Issue by bonelifer** [Operator873/fail2ban-pushover-notifications#1]
Don't quite understand option 6 wording, though. Also not sure what to do with option 5, as I'm new to fail2ban.

**Comment by Operator873** [comment]
@bonelifer You can specify specific actions for different jails. In step 4, you build the Pushover notification. Then in Step 5, you tell fail2ban to use that new action for ALL jails. If you'd rather keep the default action (no Pushover notifications) for all jails except say, sshd, you can specify actions for just that jail.

Example, open your `/etc/fail2ban/jail.local` and search for `# Choose default action.`. This will get you to the `[DEFAULT]` action section that you edit in Step 5. The default setting is `action = %(action_)s` and in Step 5 we change that to `action = %(action_ph)s`.

The optional Step 6 means you leave `action = %(action_)s` in the `[DEFAULT]` section as is and, instead, scroll down to the `[sshd]` section and under the options for that jail, add `action = %(action_ph)s`. This makes the default action just normal fail2ban jails for everything except `sshd` which would add the IP to a jail AND notifying you via Pushover.

I hope that explains it a bit better. Let me know so I can update the README if necessary.

**Comment by Operator873** [comment]
It's worth saying that the Pushover priority levels probably should be changed to match what you want.

As is, thanks to overseas activity, there is a lot of fraudulent ssh attempts happening everywhere now. So the Pushover notification can become quite noisy at times. If such is the case, to avoid alert fatigue, I'd open `pushover.sh` and change [Line 17](https://github.com/Operator873/fail2ban-pushover-notifications/blob/da49fe787770c4ba08ca9cdda4efa9b89cc86c1d/pushover.sh#L17) to `PUSHOVER_PRIORITY=-1` and [Line 22](https://github.com/Operator873/fail2ban-pushover-notifications/blob/da49fe787770c4ba08ca9cdda4efa9b89cc86c1d/pushover.sh#L22) to `PUSHOVER_PRIORITY=-2` to lower the noise level.

**Comment by bonelifer** [comment]
Yes, that explains it much more clearly. Just in the process of planning a homelab set up and gathering needed info. Thanks for for the reply.

**Comment by Operator873** [comment]
@bonelifer I'm happy to help! And welcome to the world of home-labing. It's a slippery slope so buckle in and have fun! I strongly suggest looking at Ansible for your first big learning project. Feel free to reach out to me operator873 `at` 873gear `dot` com
